presto-docs/src/main/sphinx/release/release-0.285.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.285
+=============
+
+**Details**
+===========
+
+Security Changes
+________________
+* Fix critical vulnerability in Babel and related npm packages by updating to newer versions. :pr:`21322`
```

Comment:
@tdcmeehan I felt it is easier to read the release notes if we keep them brief and include a link to the PR for more details.
Any thoughts?

Reply:
I think this is a good idea to keep the note concise, while allowing people to dig deeper.